index.rst

```diff
@@ -520,15 +578,14 @@ Naturally, people should be able to fork and customize the app for specific impl
 
 Functionality of the tool could include:
 
-- Ability to flip through a 2-d data cube as a movie
+- Ability to flip through a 2-dimensional data cube as a movie
```

Comment:
What is a "2-dimensional" cube?  Do you mean to flip through the planes of a 3-D cube, showing each one as a 2D image?

Reply:
I think the intended meaning is "a time-series of 2-dimensional scalar fields as a movie"

Reply:
Correct, except the third dimension doesn't need to be time, it can be anything (e.g. elevation).
Changed to "Ability to flip through a series 2-dimensional scalar fields as a movie, where the series represents a user-specified parameter such as time, elevation, or temperature."

Reply:
OK, cool.  Do we already have an example of this kind of data from LATISS operations?

Reply:
No, but I really want one!
I really really really want to see PSF and WFE residuals as a fxn of elevation.